benchmarks/cpp/tensorexpr/CMakeLists.txt

```diff
@@ -1,2 +1,10 @@
-add_executable(tensorexpr_bench bench_gemm.cpp bench_compile.cpp main.cpp)
+find_package(AVX)
+
+if(C_AVX2_FOUND)
+  message(STATUS "AVX2 compiler support found")
+  add_compile_options(-DUSE_AVX2)
+  add_compile_options(-mavx2)
```

Comment:
you might need to do this locally for the benchmark files (if builds break).  this comment (which might be dated) seems to indicate PT avoids setting mavx2 globally: https://github.com/pytorch/pytorch/blob/master/cmake/Dependencies.cmake#L1644-L1646

Reply:
My understanding is that add_compile_options only affect targets under the current directory and below. 

https://cmake.org/cmake/help/latest/command/add_compile_definitions.html#command:add_compile_definitions

I will try to be a bit more specific here. Thanks!